### DISCUSSION TITLE
Seeking help in learning tikz-feynman package

### DISCUSSION THREAD

**Discussion by topsquarks** [community/community#34468] [Actions]
If I'm in the wrong place someone please let me know where I need to go.

I am having some troubles with the tikz-feynman package.  I'm just starting out and it's likely a simple coding issue.  I am trying to create a basic horizontal four point diagram which is essentially a tree diagram with a fermion loop in the middle.  I can't get the incoming and outgoing fermion lines to work right.  I will provide my code once I know I'm in the right place to ask for coding help.  

Thank you!

-Dan

**Comment by airtower-luna** [comment]
I'm not sure if you have the right place. The GitHub Community is primarily for discussions around Git and GitHub. Coding questions are allowed (though AFAIK they should go into the "general" category), but your topic seems so specialized I'm not sure if you'll get much of a response here. A place more focused on scientific writing and LaTeX might work better.

**Comment by topsquarks** [comment]
Okay.  I wasn't sure.  Thanks!

-Dan
